react-carousel/test/plugins/infinite.test.js

```diff
@@ -0,0 +1,11 @@
+import setupCarousel from '../tools/setupCarousel';
+
+describe('infinite', () => {
+  it('renders additional clones when in infinite mode', () => {
+    const wrapper = setupCarousel({
+      plugins: ['infinite'],
+    });
+
+    expect(wrapper.find('.BrainhubCarouselItem')).toHaveLength(15);
```

Comment:
why 15?

Reply:
From this equation
```
numberOfSlides + (numberOfSlides * numberOfClones * numberOfSides)
```
For this test numberOfSlides equals 3
numberOfClones wasn't specified and by the default it's 2
numberOfSides always equals two because we making copies for the left side of original items and for the right side

Reply:
- it would be good to explain that in a comment
- it would be good to set up explicitly to see the number of slides and the number of clones without a need of jumping into another file